packages/api/cms-api/src/user-permissions/content-scope.service.ts

```diff
@@ -82,7 +82,7 @@ export class ContentScopeService {
                 }
 
                 for (const id of ids) {
-                    const row = await repo.findOneOrFail(id);
+                    const row = await repo.findOneOrFail(id, { filters: false });
```

Comment:
maybe add a comment like
```suggestion
                    const row = await repo.findOneOrFail(id, { filters: false }); // disable all default filters, like eg. excludeDeleted
```